docs/community/contribute/contribute_code.rst

```diff
@@ -25,12 +25,38 @@ To understand how the below components interact with each other, refer to :ref:`
     * - **Purpose**: Deployment, Documentation, and Issues
     * - **Languages**: RST
 
-To build the Flyte docs locally you will need the following prerequisites:
+In the ``flyteorg/flyte`` root directory you can run ``make dev-docs`` to build the documentation locally. The generated documentation will be in the ``docs/_build/html`` directory.
 
+**Setup process**
+
+1. First you need to make sure you can run linux/amd64 container
```

Comment:
Can arm macs by default run amd64 images?  Not well right?  I feel like I get errors all the time.  Is the image multi-arch?